src/Model/UserModel.ts

```diff
@@ -0,0 +1,144 @@
+import { router } from "yandex-maps";
+import { apiConfig } from "../modules/api";
+import { Api } from "../modules/api/src/api";
+import { EventDispatcher, Listenable } from "../modules/observer";
+
+export type User = {
+    Username: string;
+    Email: string;
+    Password: string;
+    Icon: string;
+};
```

Comment:
Интерфейсы являются более предпочтительным синтаксисом

```suggestion
export interface User {
    Username: string;
    Email: string;
    Password: string;
    Icon: string;
}
```